### PR TITLE
Gallery implementation

### PR DESCRIPTION
**Pull request by aannie-y**
Grid gallery implementation. This code ONLY contains the gallery section and its corresponding css. Code has been checked with clang formatter.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,45 @@
...
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="galleryDiv">
+        <h1>Gallery</h1>
+        <div class="gallery">
+          <a target="_blank" href="images/remarkables.jpg">
+            <img src="images/remarkables.jpg" width="600" height="400">
```
Comment (KathleenQ) [inline portfolio/src/main/webapp/index.html]: As Lilian's previous advice for me, it is better to split width and height out as separate style requirement for img in "style.css":
.img{width = ...; ...}

**Review by KathleenQ — COMMENTED** [review]
Format looks good after the check.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (aannie-y) [reply portfolio/src/main/webapp/index.html]: ok will do. Thanks!

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -53,4 +53,53 @@
...
+  padding-left: 200px;
+  padding-right: 200px;
+}
+.gallery {
+  margin: 5px;
+  border:1px solid #ccc;
+  float: left;
+  width: 250px;
+  height: 230px;
+  /* display: flex; */
```
Comment (tripd22) [inline portfolio/src/main/webapp/style.css]: Nit: Please remove if you don't want to use this

**Review by jupi3200 — APPROVED** [review]
LGTM

**Pull request merged**